src/shared/regions/regionUtilities.ts

```diff
@@ -0,0 +1,18 @@
+/*!
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { AwsContext } from '../awsContext'
+import { Region } from './endpoints'
+import { RegionProvider } from './regionProvider'
+
+const DEFAULT_REGION = 'us-east-1'
+const DEFAULT_PARTITION = 'aws'
+
+export function getRegionsForActiveCredentials(awsContext: AwsContext, regionProvider: RegionProvider): Region[] {
```

Comment:
could go in AwsContext

Reply:
This was tempting, but since AwsContext is an interface, it felt like more of a utility method than an implementation detail.